src/components/ArchivedReportFooter.js

```diff
@@ -74,6 +74,7 @@ const ArchivedReportFooter = (props) => {
             })}
             shouldRenderHTML={archiveReason !== CONST.REPORT.ARCHIVE_REASON.DEFAULT}
             shouldShowIcon
+            textStyles={styles.pre}
```

Comment:
We won't need this right?

Reply:
That right. Thanks for pointing out that